cluster_experiments/analysis_plan.py

```diff
@@ -0,0 +1,264 @@
+from typing import List
+
+import pandas as pd
+from pandas import DataFrame
+
+from cluster_experiments.analysis_results import (
+    AnalysisPlanResults,
+    HypothesisTestResults,
+)
+from cluster_experiments.hypothesis_test import HypothesisTest
+from cluster_experiments.power_config import analysis_mapping
+from cluster_experiments.variant import Variant
+
+
+class AnalysisPlan:
+    """
+    A class used to represent an Analysis Plan with a list of hypothesis tests and a list of variants.
```

Comment:
I'd mention that what is specific about analysis plan is that all hypotheses tests inside it can be run with the same dataframe

Reply:
true!! will add it